client/src/components/Game/Endgame.js

```diff
@@ -53,20 +71,23 @@ class Endgame extends Component {
 
     // Wait for blobs to finish fetching and add them to a zip folder
     Promise.all(blobs);
-    blobs.forEach((blob, i) => folder.file(players[i] + '.png', blob, { binary: true }));
+    blobs.forEach((blob, i) => folder.file(i+1 + ' - ' + usernames[i] + '.png', blob, { binary: true }));
```

Comment:
[String literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) might be nice for this!

Reply:
ty forgot about those, will fix